astropy/coordinates/builtin_frames/astrometric.py

```diff
@@ -124,9 +123,9 @@ def astrometric_to_astrometric(from_astrometric_coord, to_astrometric_frame):
         # Otherwise, the transform occurs just by setting the new origin.
         return to_astrometric_frame.realize_frame(from_astrometric_coord.cartesian)
 
-    @frame_transform_graph.transform(DynamicMatrixTransform, framecls, Astrometric)
+    @frame_transform_graph.transform(DynamicMatrixTransform, framecls, _Astrometric)
     def icrs_to_astrometric(reference_frame, astrometric_frame):
```

Comment:
Change `icrs_to_astrometric` to `reference_to_astrometric` here?
This was a suggestion @taldcroft made in the other issue and I also found this confusing to name this `icrs` when it's not specific to icrs.